src/bci_build/package/__init__.py

```diff
@@ -348,8 +348,8 @@ def __post_init__(self) -> None:
             # Limit to aarch64 and x86_64
             if not self.exclusive_arch:
                 self.exclusive_arch = [Arch.AARCH64, Arch.X86_64]
-            # Disable maintainer listing
-            self.maintainer = None
+            # Disable maintainer listing from base container
```

Comment:
Maybe this is a bit clearer?
```suggestion
            # List maintainer as empty string to ensure that base container value is overwritten
```

Reply:
updated.